File: 백준/Bronze/29738. Goodbye， Code Jam/Goodbye， Code Jam.py
def check_round(num: int):
    if num <= 25:
        return "World Finals"
    if num <= 1000:
        return "Round 3"
    if num <= 4500:
        return "Round 2"
    return "Round 1"

t = int(input())
for i in range(1, t + 1):
    num = int(input())
    print(f"Case #{i}: {check_round(num)}")

File: 백준/Bronze/29738. Goodbye， Code Jam/Goodbye， Code Jam.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        for (int i = 1; i <= t; i++) {
            int num = scanner.nextInt();
            System.out.println("Case #" + i + ": " + checkRound(num));
        }
        scanner.close();
    }

    public static String checkRound(int num) {
        if (num <= 25) {
            return "World Finals";
        }
        if (num <= 1000) {
            return "Round 3";
        }
        if (num <= 4500) {
            return "Round 2";
        }
        return "Round 1";
    }
}